05_iigo/index.tex

```diff
@@ -347,52 +348,69 @@ \section{Accountability Cycle}
 
 \subsection{Transfer-of-power}
 \label{subsec:transfer-of-power}
+
+For the scope of this project, we chose elections to be the only system of power transfer for the islands to utilise. The islands that hold institutional power are the decision group of the archipelago. They decide on taxes, allocations and sanctions. By holding an election for the institutional roles, the islands are not directly included in the decision group, but they do participate in deciding who will occupy these roles and thus who makes the aforementioned decisions. Elections also open up another avenue for opinion formation to have an effect.
+\begin{itemize}
+    \item Each role has the \emph{power} to call an election vote and declare the winner (see Figure~ \ref{fig:cycles_in_IIGO} for the power transfer cycle).
+\end{itemize}
+
+We note that:
 \begin{enumerate}
-    \item The Speaker conducts a vote for the election of the new Judge.
-    \item The Judge conducts a vote for the election of the new President.
-    \item The President conducts a vote for the election of the new Speaker.
+    \item The Speaker conducts an election to appoint a new Judge.
+    \item The Judge conducts an election to appoint a new President.
+    \item The President conducts an election to appoint a new Speaker.
 \end{enumerate}
 Refer to the Figure~ \ref{fig:cycles_in_IIGO} for further clarification about the transfer-of-power cycle.
 
-\begin{rule_IIGO}
-   The role $X$ has the \emph{obligation} to conduct a vote for the election of $Y$ at the end of the term (see Definition~\ref{def:term}) if and only if the role $X$ is accountable to the role $Y$. 
-\end{rule_IIGO}.
+We introduce a \emph{term} length to increase the diversity of the decision group. If the Rule~\ref{rule:roles_must_hold_election} is in play, the roles are obliged to hold an election every $N$ turns. To reduce the scope of the coursework, the term length is defined as a configuration parameter. Thus we reduce the complexity of rules surrounding the election of roles and hence the reasoning the agents have to do with regard to said rules.
```

Comment:
```suggestion
We introduce a \emph{term} length to increase the diversity of the decision group. If the Rule~\ref{rule:roles_must_hold_election} is in play, the roles are obliged to hold an election every $N$ turns. To reduce the scope of the coursework, the term length is defined as a configuration parameter. Thus, we reduce the complexity of rules surrounding the election of roles and also the reasoning the agents have to do with this regard to the specified rules.
```

Reply:
```suggestion
We introduce a \emph{term} length to increase the diversity of the decision group. If the Rule~\ref{rule:roles_must_hold_election} is in play, the roles are obliged to hold an election every $N$ turns. To reduce the scope of the coursework, the term length is defined as a configuration parameter. Thus, we reduce the complexity of rules surrounding the election of roles and hence the reasoning the agents have to do with regard to said rules.
```

Agents have less to reason about as a result of simpler rules, hence the word "hence" is appropriate

Reply:
ok what is 'said' rules?

Reply:
rules surrounding the election

Reply:
I don't think we can call them 'said' rules ... rewording is needed

Reply:
```suggestion
We introduce a \emph{term} length to increase the diversity of the decision group. If the Rule~\ref{rule:roles_must_hold_election} is in play, the roles are obliged to hold an election every $N$ turns. To reduce the scope of the coursework, the term length is defined as a configuration parameter. Thus, we reduce the complexity of rules surrounding the election of roles and hence the reasoning the agents have to do with regard to these rules.
```